use-timescale/data-tiering/index.md

```diff
@@ -53,11 +57,26 @@ In this section, you:
 * [Learn about replicas and forks with tiered data][replicas-and-forks]: understand how tiered storage works
   with forks and replicas of your $SERVICE_SHORT.
 
+</Tab>
+
+<Tab title="Tiger Cloud on Azure" label="azure-cloud">
+
+$CLOUD_LONG stores your data in high-performance storage optimized for frequent querying. Based on [AWS EBS gp3][aws-gp3], the high-performance storage provides you with up to 16 TB and 16,000 IOPS. Its [$HYPERCORE row-columnar storage engine][hypercore], designed specifically for real-time analytics, enables you to compress your data by up to 98%, while improving performance. 
```

Comment:
With the recent changes in gp3, which now supports 64000 IOPS and 64TB, we need @rahilsondhi to reconsider external communications about it

Reply:
$CLOUD_LONGs [$HYPERCORE ...